InstallFullDB.sh

```diff
@@ -1470,10 +1470,10 @@ function apply_dev_content
 # Apply playerbot sql files
 function apply_playerbots_db
 {
-  if [ "$PLAYERBOTS_DB" != "YES" ]; then
-    true
-    return
-  fi
+  # if [ "$PLAYERBOTS_DB" != "YES" ]; then
+  #   true
+  #   return
+  # fi
```

Comment:
this is obviously not ideal, but I also don't have any command line options with which I can force the bots to be built